src/Servers/IIS/AspNetCoreModuleV2/CommonLibTests/CommonLibTests.vcxproj

```diff
@@ -2,7 +2,8 @@
   <PropertyGroup>
     <BuildHelixPayload>false</BuildHelixPayload>
   </PropertyGroup>
-  <ItemGroup Label="ProjectConfigurations">
+
+ <ItemGroup Label="ProjectConfigurations">
```

Comment:
nit: Misaligned indentation❔

Reply:
Is there a common tab pattern we are supposed to use for msbuild stuff?  I noticed the settings spacing was slightly different than the vcxproj, I just tried to keep the spacing the same in each file rather than normalizing

Reply:
We generally follow regular XML nesting w/ two spaces per indentation level and a fair amount of randomness for attribute indentation if lines get too long